docs/contributing/tasks.md

```diff
@@ -1,31 +1,31 @@
-# NPM and Gulp tasks
+# npm and Gulp tasks
 
-This document describes the NPM scripts that run the application, and the gulp tasks they trigger to build files, update the package, copy assets and watch for changes.
+This document describes the npm scripts that run the application, and the gulp tasks they trigger to build files, update the package, copy assets and watch for changes.
 
 To run the application without any tasks being triggered, see [Express app only](#express-app-only).
 
-## NPM script aliases
+## npm script aliases
 
-NPM scripts are defined in `package.json`. These trigger a number of gulp tasks.
+npm scripts are defined in `package.json`. These trigger a number of gulp tasks.
 
 **`npm run start` will trigger `gulp dev` that:**
 - cleans the `public` folder
-- compiles component nunjucks files to `public`
+- compiles component Nunjucks files to `public`
 - copies icons to `public`
-- compile sass files, add vendor prefixes and copy to `public`
+- compile Sass files, add vendor prefixes and copy to `public`
 - starts up the Express server and app
 - starts up `gulp watch` task to watch for changes
 
 **`npm run test` will do the following:**
 - compile components to HTML
-- run JS tests
-- run CSS lint checker
+- run JavaScript tests
+- run Sass lint checker
 - run accessibility tests on HTML files
 - run tests on the review application
 
 **`npm run heroku` runs on Heroku build/PR and it:**
 - compiles components' HTML
-- compiles CSS & JS
+- compiles Sass & JavaScript
```

Comment:
Thanks for this @domoscargin

The diff has changed but I've gone with:

### **Short descriptions**
For npm scripts (or gulp tasks with sub tasks)
```markdown
**`npm run heroku` runs on Heroku build/PR and it will:**
- run `npm run build:compile`
- start up Express
```

### **Long descriptions**
For the main gulp task that does the work
```markdown
**`gulp styles`**

This task will:
 - check Sass code quality via Stylelint (`npm run lint:scss`)
 - compile Sass to CSS (`gulp scss:compile`) into `./public`, or another location via the `--destination` flag

**`gulp scripts`**

This task will:
 - check JavaScript code quality via ESLint (`npm run lint:js`) (using JavaScript Standard Style)
 - compile JavaScript ESM to CommonJS (`gulp js:compile`) into `./public`, or another location via the `--destination` flag
```

Is that alright?